next-sample-pages-router/src/pages/_app.tsx

```diff
@@ -27,7 +27,7 @@ export default function App({ Component, pageProps }: AppProps) {
         <Link href="/static-dashboard">Static Dashboard</Link>
         <Link href="/interactive-dashboard">Interactive Dashboard</Link>
       </nav>
-      <MetabaseProvider config={config}>
+      <MetabaseProvider authConfig={config}>
```

Comment:
let's rename this and the rest of the instance `authConfig` as well, as we also use those in the readme.

```suggestion
      <MetabaseProvider authConfig={authConfig}>
```

https://github.com/metabase/metabase/blob/e3f52a4e995631da8956f25612231e155aa28f75/docs/embedding/sdk/authentication.md#L176

Reply:
Done, thank you